packages/components-stencil/src/components/alert/readme.md

```diff
@@ -0,0 +1,20 @@
+# t-button
```

Comment:
I am bit confused why the headline is called t-button but the markdown file is placed within the alert directory

Reply:
fixed